static/app/components/events/contexts/contextIcon.tsx

```diff
@@ -138,18 +138,24 @@ function getLogoImage(name: string) {
 
 type Props = {
   name: string;
+  hideUnknown?: boolean;
   size?: IconSize;
 };
 
-function ContextIcon({name, size: providedSize = 'xl'}: Props) {
+function ContextIcon({name, size: providedSize = 'xl', hideUnknown = false}: Props) {
   const theme = useTheme();
   const size = theme.iconSizes[providedSize];
 
   // Apply darkmode CSS to icon when in darkmode
   const isDarkmode = useLegacyStore(ConfigStore).theme === 'dark';
   const extraCass = isDarkmode && INVERT_IN_DARKMODE.includes(name) ? darkCss : null;
 
-  return <img height={size} width={size} css={extraCass} src={getLogoImage(name)} />;
+  const imageName = getLogoImage(name);
+  if (hideUnknown && imageName === logoUnknown) {
+    return null;
+  }
+
+  return <img height={size} width={size} css={extraCass} src={imageName} />;
```

Comment:
Could wrap in our `Tooltip` component if you want a non-native tooltip

Reply:
Oh yeah that would have been better, good call! But [we opted](https://sentry.slack.com/archives/C06L5GXMU8H/p1717458595000159) to remove the title/tip altogether, but I'll keep it in mind for next time